doc/deployments/secrets.md

```diff
@@ -0,0 +1,19 @@
+# Secrets
+
+## Adding secrets
```

Comment:
Long term we should have some kind of capability assignment for resources we control. But we can think about this over time.